src/traces/scatter/attributes.js

```diff
@@ -292,14 +292,27 @@ module.exports = {
             ].join(' ')
         },
         dash: extendFlat({}, dash, {editType: 'style'}),
+        backoff: { // TODO: do we want to have a similar option for the start of the line? If so what the name should be?
+            valType: 'number',
+            min: 0,
+            dflt: 'auto',
+            arrayOk: true,
+            editType: 'plot',
+            description: [
+                'Sets the line back off from the end point of the nth line segment (in px).',
+                'This option is useful e.g. to avoid overlap with arrowhead markers.',
+                'With *auto* the lines would trim before markers if `marker.angleref` is set to *previous*.'
+            ].join(' ')
+        },
         simplify: {
             valType: 'boolean',
-            dflt: true,
             editType: 'plot',
             description: [
                 'Simplifies lines by removing nearly-collinear points. When transitioning',
                 'lines, it may be desirable to disable this so that the number of points',
-                'along the resulting SVG path is unaffected.'
+                'along the resulting SVG path is unaffected.',
+                'Defaults to *false* when `backoff` is set,',
```

Comment:
Is this because simplify + backoff throws errors (in which case `false` should not only be the default, but the only allowed value in that case) or just that in some cases it has unexpected results (in which case flipping the default is appropriate but we should mention the reason in this description)?